src/components/RepoItem/RepoItem.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import styles from "../RepoItem/RepoItem.module.css";
+import classnames from "classnames";
+
+const RepoItem = ({
+  url,
+  name,
+  language,
+  stargazers_count,
+  forks_count,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'forks_count' is not in camel case.](https://app.codacy.com/gh/WebHeroSchool/tododo/pullRequest?prid=5504860)